apps/Test App/Test App/Widgets/UploadCTA.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  BigUploadCta.swift
+//  Test App
+//
+//  Created by Emily Dixon on 5/11/23.
+//
+
+import SwiftUI
+import PhotosUI
+
+struct BigUploadCTA: View {
```

Comment:
I renamed this to `BigUploadCTALabel`, since that's really what it is. I tried doing it the way you described, but it wound up not working out. Gonna resolve this one if you don't mind

Reply:
Totally, thanks for looking into it!